app/controllers/bookingsync_portal/admin/remote_accounts_controller.rb

```diff
@@ -10,7 +10,7 @@ def new
       def create
         BookingsyncPortal::Write::EnsureSourceExists.new(current_account).call
         @remote_account = scope.create(params_remote_account)
-        respond_with @remote_account, location: admin_rentals_url
+        respond_with @remote_account, location: admin_rentals_path
```

Comment:
in rails 5 the host is not automatically inferred from env, so those don't work with extra setup.
but it is not necessary to have full urls here, paths should be good enough?